File: Lab1/Lab1/stove.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lab1
{
    class stove
    {
        private skovoroda skovorods;

        public bool State { set; get; }

        public skovoroda Skov { set { skovorods = value; } get { return skovorods; } }

        public void Cook()
        {
            if (State)
            {
                while (skovorods.IsReady() == false)
                {
                    skovorods.GetHeat();
                }
            }
        }


    }
}

File: Lab1J/src/stove.java

public class stove {
	private skovoroda skovorods;
	
	private boolean State = false;
    public void setState(boolean m){
    	State = m;
    }
    
    public boolean getState(){
    	return State;
    }
    
    public void setScovorods(skovoroda value){
    	skovorods = value;
    }
    
    public skovoroda getScovorods(){
    	return skovorods;
    }

    public void Cook()
    {
        if (State)
        {
            while (skovorods.IsReady() == false)
            {
                skovorods.GetHeat();
            }
        }
    }
}
